src/cloud-api-adaptor/ibmcloud/image/login.sh

```diff
@@ -26,7 +26,9 @@ if ! ibmcloud iam oauth-tokens &> /dev/null; then
 fi
 
 required_plugins=(vpc-infrastructure cloud-object-storage)
-mapfile -t installed_plugins < <(ibmcloud plugin list --output json | jq -r '.[].Name')
+# mapfile -t installed_plugins < <(ibmcloud plugin list --output json | jq -r '.[].Name')
+installed_plugins=($(ibmcloud plugin list --output json | jq -r '.[].Name'))
+# ^^^ mapfile is not available on mac and is not needed here anyway
```

Comment:
Hi Frank, you have remove the comments about mapfile here I think - the code fix should be sufficient